src/ray/rpc/client_call.h

```diff
@@ -30,6 +30,8 @@ class ClientCall {
   /// The callback to be called by `ClientCallManager` when the reply of this request is
   /// received.
   virtual void OnReplyReceived() = 0;
+  /// Return status.
+  virtual ray::Status GetStatus() = 0;
```

Comment:
Is this used?

Reply:
yes, it's used [here](https://github.com/ant-tech-alliance/ray/blob/add_grpc_server_to_worker/src/ray/rpc/worker/worker_client.h#L44).  This is similar to the async write callback with asio, we want to check if we succeed on assigning a specific task to a worker.